### ISSUE TITLE
Transaction failed on Sepolia using Simple/PasskeyAccount v0.7.0

### ISSUE THREAD

**Issue by oneleo** [pilagod/waallet-extension#140]
## Issue in
- commit: [1b81ae35905f030460caa69d97516678ff870138](https://github.com/pilagod/waallet-extension/tree/1b81ae35905f030460caa69d97516678ff870138)

### 1. SimpleAccount v0.7.0
- Using [SimpleAccount v0.7.0](https://github.com/eth-infinitism/account-abstraction/blob/v0.7.0/contracts/samples/SimpleAccount.sol) with [VerifyingPaymaster v0.7.0](https://github.com/eth-infinitism/account-abstraction/blob/v0.7.0/contracts/samples/VerifyingPaymaster.sol) on Sepolia, Alchemy Bundler returns this error:

```
[JsonRpcProvider][eth_sendUserOperation][response] {
  "jsonrpc": "2.0",
  "id": 0,
  "error": {
    "code": -32000,
    "message": "precheck failed: preVerificationGas is 45060 but must be at least 46824"
  }
}
```

### 2. PasskeyAccount v0.7.0
- Using [PasskeyAccount v0.7.0](https://github.com/pilagod/waallet-contract/blob/main/src/account/0.7/PasskeyAccount.sol) on Sepolia, Alchemy Bundler returns [this error](https://docs.alchemy.com/reference/error-reference#json-rpc-error-codes):

```
.[JsonRpcProvider][eth_sendUserOperation][response] {
  "jsonrpc": "2.0",
  "id": 0,
  "error": {
    "code": -32603,
    "message": "(code: -32000, message: execution timeout at Integer (bigInt:1:158(0)) at t (<eval>:1:1608(36))    in server-side tracer function 'step', data: None)"
  }
}
```

**Comment by pilagod** [comment]
This first problem is solved by this commit [b2bf546](https://github.com/pilagod/waallet-extension/pull/137/commits/b2bf54615cb4c94f577e9a9d6445885af10093b3)

**Comment by pilagod** [comment]
The timeout problem can be mitigated on chain with precompiled p256 verifier, see #144.
